pkg/web/static/js/transactionAccept.js

```diff
@@ -0,0 +1,4 @@
+// Get the transaction ID from the URL and add it to the htmx request.
+const id = window.location.pathname.split('/').pop()
+document.getElementById('transaction-accept').setAttribute('hx-get', `/v1/transactions/${id}/accept`)
+
```

Comment:
Wouldn't it be better to compute this URL in the view (e.g. using a Go method) rather than with Javascript like this? I think that would be more maintainable in the long run.